py3status/modules/gpmdp.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Display currently playing song from Google Play Music Desktop Player.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh this module in seconds (5s default)
+
+Requires
+    - gpmdp
+    - gpmdp-remote
+
+@author Aaron Fields https://twitter.com/spirotot
+@license BSD
+"""
+
+# import your useful libs here
+from time import time
+from subprocess import check_output
+import shlex
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+
+    cache_timeout = 5
+    format = '{info}'
+
+    def gpmdp(self, i3s_output_list, i3s_config):
+        result = u'♫ ' + self.format
+
+        if check_output(shlex.split('gpmdp-remote status')).decode('ascii', 'ignore').strip() == 'Paused':
+            result = ''
+        else:
+            cmds = ['info', 'title', 'artist', 'album', 'status', 'current',
+                    'status', 'time_total', 'time_current', 'album_art']
+
+            for cmd in cmds:
+                if str('{' + cmd + '}') in result:
+                    result = result.replace(str('{' + cmd + '}'),check_output(shlex.split('gpmdp-remote %s' % (cmd))).decode('ascii', 'ignore').strip())
```

Comment:
Just pushed changes, using `self.format.format()`. Let me know if there's anything else you'd like to see changed.